alarm/Alarm.js

```diff
@@ -347,7 +347,7 @@ class VPNRestoreAlarm extends Alarm {
   }
 
   localizedNotificationContentArray() {
-    return [this["p.vpn.subtype"], this["p.vpn.devicecount"], this["p.vpn.displayname"], this["p.vpn.time"], this['p.vpn.subtypename'], this["p.vpn.strictvpn"]];
+    return [this["p.vpn.subtype"], this["p.vpn.devicecount"], this["p.vpn.displayname"], this['p.vpn.subtypename'], this["p.vpn.strictvpn"]];
```

Comment:
if we are still going to show time in notification, we should still add timestamp here in the array